packages/plugin-ext-vscode/src/common/plugin-vscode-types.ts

```diff
@@ -14,4 +14,4 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-export const VSCODE_DEFAULT_API_VERSION = '1.50.0';
+export const VSCODE_DEFAULT_API_VERSION = '1.52.0';
```

Comment:
Should we not aim for better compatibility first before targeting newer versions of extensions?

Reply:
@vince-fugnitto As far as I understand this value doesn't make much sense or maybe I'm wrong. Could you please explain, how can we know that it has good enough compatibility

Reply:
> @vince-fugnitto As far as I understand this value doesn't make much sense or maybe I'm wrong. Could you please explain, how can we know that it has good enough compatibility

@svor this value is used to specify which default API we support, without it we would always fetch `latest` from the open-vsx registry through the `extensions` view and potentially install extensions which are not supported by the framework. We should only update the default API version when we are confident that we support the latest APIs.

Some big items to consider:
- upgrading electron
- upgrading to the latest monaco version
- customExecution API

Other APIs defined by the compatibility chart:
- https://che-incubator.github.io/vscode-theia-comparator/status.html

Reply:
@vince-fugnitto Got it, thanks!
I've reverted changes related to `VSCODE_DEFAULT_API_VERSION`